Add Amazon Linux, SUSE SLES, and openSUSE Leap Linux distro support to ec2_info module (#22202)
hashicorp/vault: website/content/docs/configuration/storage/raft.mdx

- `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
  This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.

> - `performance_multiplier` `(integer: 0)` - An integer multiplier used by 
>   servers to scale key Raft timing parameters, where each increment translates to approximately 1 – 2 seconds of delay. For example, setting the multiplier to "3" translates to 3 – 6 seconds of total delay.  Tuning  the multiplier affects the time it

Reviewer: ```suggestion
- `performance_multiplier` `(integer: 0)` - An integer multiplier used by 
  servers to scale key Raft timing parameters. Each scalling increment
  translates to approximately 1 – 2 seconds of delay. For example, setting the
  multiplier to "3" translates to 3 – 6 seconds of total delay. Tuning the multiplier affects the time it
```

Style correction: 1 idea per sentence.

  takes Vault to detect leader failures and to perform leader elections, at the
  expense of requiring more network and CPU resources for better performance.

>   Omitting this value or setting it to 0 uses default timing described below. 

Reviewer: ```suggestion
  Omitting the performance multiplier or setting it to 0 uses default timing described below. 
```
Style correction: avoid "this" as a pronoun
